tests/foreman/cli/test_repository.py

```diff
@@ -480,33 +411,29 @@ def test_positive_update_4(self, checksum_type):
         @BZ: 1208305
 
         """
-        content_type = u'yum'
-        repository = self._make_repository({u'content-type': content_type})
-        self.assertEqual(repository['content-type'], content_type)
-        self.assertEqual(repository['checksum-type'], '')
-        # Update the url
-        Repository.update({
-            u'checksum-type': checksum_type,
-            u'id': repository['id'],
-        })
-        # Fetch it again
-        result = Repository.info({'id': repository['id']})
-        self.assertNotEqual(
-            result['checksum-type'],
-            repository['checksum-type'],
-        )
-        self.assertEqual(result['checksum-type'], checksum_type)
+        for checksum_type in u'sha1', u'sha256':
+            with self.subTest(checksum_type):
+                content_type = u'yum'
+                repository = self._make_repository({
+                    u'content-type': content_type
+                })
+                self.assertEqual(repository['content-type'], content_type)
+                self.assertEqual(repository['checksum-type'], '')
```

Comment:
if the purpose of the test is truly to incorporate the creation, then this is fine. If it is more so to check the update, then this logic could be placed about the loop.